color_refinement.py

```diff
@@ -141,7 +122,7 @@ def is_isomorphisms(g: Graph, h: Graph) -> bool:
 def get_number_automorphisms(g: Graph) -> int:
     """
     Returns the number of isomorphisms of graph g
-
+P
```

Comment:
💔

Reply:
Nu kan ik net zo goed ophouden met software schrijven. Ik wordt wel stratenmaker.

Reply:
@jelmerterwal

Reply:
haha Erik, leuk.

Reply:
maar deze review is iig gefixt